install-deps.sh

```diff
@@ -395,7 +395,14 @@ else
     debian|ubuntu|devuan|elementary|softiron)
         echo "Using apt-get to install dependencies"
         if [ "$INSTALL_EXTRA_PACKAGES" ]; then
-            $SUDO apt-get install -y $INSTALL_EXTRA_PACKAGES
+            if ! $SUDO apt-get install -y $INSTALL_EXTRA_PACKAGES ; then
+                # try again. ported over from run-make.sh. Not sure why this is
```

Comment:
there is chance that, for instance, `apt-get` is interrupted when installing / removing packages, in that case, the package management system would be in an inconsistent state. if we run `apt-get install ...` when the system is in this state, the tool would prompt us to run `dpkg --configure -a`.  i think the commit message of e2782954088dadb9bfe9cd3760129f06c602e7dd does explain this .

Reply:
Thanks! I'll update the comment to match.

Reply:
Updated.